showcase/app/styles/showcase-components/frame.scss

```diff
@@ -6,9 +6,9 @@
 
 $shw-frame-navigation-bar-height: 48px;
 
-.shw-frame {
-  margin-bottom: 2rem;
-}
+// .shw-frame {
+//   margin-bottom: 2rem;
+// }
```

Comment:
I updated AppFrame Showcase to use frame examples inside `Shw::Grid` so this style was adding an extra margin I didn't want/need. Would it be ok to delete this style?

Reply:
When I implemented the `Shw::Frame` I imagined they would always be used in isolation, because of their nature, and because I would not imagine them used side by side. 
Hence the margin bottom. 

Why do you want to use them in a grid? Because of the label? In that case `Shw::Frame` supports a `@label` argument (as well as `@width` and `@height` if needed)

Reply:
Ok I'll change it back. I just thought it seemed redundant since we already had components and patterns for adding labels & spacing